src/Service/Reports/SaCoreTypesReportService.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace Linn.LinnappsUi.Service.Reports
+{
+    using System.Collections.Generic;
+    using System.Data;
+
+    using Linn.Common.Configuration;
+    using Linn.LinnappsUi.Domain.ReportModels;
+    using Linn.LinnappsUi.Domain.Reports;
+
+    using Oracle.ManagedDataAccess.Client;
+
+    public class SaCoreTypesReportService : ISaCoreTypesReportService
+    {
+        public IEnumerable<SalesArticleCoreType> GetCoreTypesBySalesArticle()
+        {
+            var connection = new OracleConnection(this.GetConnectionString());
+
+            var sql =
```

Comment:
Is there any way we can avoid embedding SQL statements in code? Not a nice way to handle DB interactions

Reply:
yeah. we don't have to do any part of this. Just playing about but it is potentially a very quick way of replacing reports with complex sql